Add support for Build Tool v28 and Gradle 3.2.1 to travis.yml file

**Describe the chore**
The latest version of Android studio is incompatible with the Gradle version used in this project (gradle:3.2.0-alpha16). Whenever the project is imported into Android Studio, the developers are presented with this error:

![Error Screenshot](https://user-images.githubusercontent.com/24315306/47568711-7433dd80-d94f-11e8-8bbe-84d71faf4394.png)

If the developer updates the Gradle version to 3.2.1 and creates commits to this repository, the Travis CI builds for those commit fail with the following error:

```
FAILURE: Build failed with an exception.
* What went wrong:
A problem occurred configuring project ':app'.
> Failed to install the following Android SDK packages as some licences have not been accepted.
     build-tools;28.0.3 Android SDK Build-Tools 28.0.3
  To build this project, accept the SDK license agreements and install the missing components using the Android Studio SDK Manager.
  Alternatively, to transfer the license agreements from one workstation to another, see http://d.android.com/r/studio-ui/export-licenses.html
```

This error occurs because the current travis.yml file doesn't have support for build tools v28.


**Would you like to work on the issue?**
- [x] Yes

--- a/.travis.yml
+++ b/.travis.yml
@@ -23,3 +23,5 @@ script:
 after_success:
     - bash scripts/prep-key.sh
     - bash scripts/update-apk.sh
+before_install:
+- yes | sdkmanager "platforms;android-28"

--- a/build.gradle
+++ b/build.gradle
@@ -8,7 +8,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.0-alpha16'
+        classpath 'com.android.tools.build:gradle:3.2.1'
         classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
 
         // NOTE: Do not place your application dependencies here; they belong

